sentry/map.jinja

```diff
@@ -0,0 +1,27 @@
+{%- load_yaml as defaults %}
+Debian:
+  pkgs:
```

Comment:
there are no multiple methods now, when is it can be upgraded with pip as default, I'd move to issues